packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -86,13 +103,57 @@ export class KeymapsFrontendContribution extends AbstractViewContribution<Keybin
             isVisible: w => this.withWidget(w, () => true),
             execute: w => this.withWidget(w, widget => widget.clearSearch()),
         });
+        commands.registerCommand(KeymapsCommands.COPY_COMMAND, {
+            execute: ({ id, value }: { id: string, value: KeybindingItem }) => {
+                this.clipboardService.writeText(this.toStringifiedCommand(value));
+            }
+        });
+        commands.registerCommand(KeymapsCommands.COPY_COMMAND_ID, {
+            execute: ({ id }: { id: string }) => {
+                this.clipboardService.writeText(id);
+            }
+        });
+        commands.registerCommand(KeymapsCommands.EDIT_KEYBINDING, {
+            execute: async ({ id, value }: { id: string, value: KeybindingItem }) => {
+                const widget = await this.widget;
```

Comment:
`this.widget` create a new widget even it it is not visible, why don't use `withWidget`?

Reply:
I wasn't sure how to use both `withWidget` and passing the values (`{ id, value }`) from the context-menu to the command simultaneously.

Reply:
> `this.widget` create a new widget even it it is not visible, why don't use `withWidget`?

I wasn't sure how to use both `withWidget` and the passed values from the context-menu
 `{ id, value }` simultaneously.